lib/datadog/tracing/contrib/aws/instrumentation.rb

```diff
@@ -40,6 +40,13 @@ def annotate!(span, context)
               handler.add_tags(span, params)
             end
 
+            if configuration[:peer_service]
```

Comment:
👍 . You could probably collapse this to a line if you wanted to; doesn't matter much though.

Reply:
I think for now we can keep it the way it is and mass refactor if we need to, but I also agree it's fine the way it is now